packages/applicant/src/pages/submissions/[submissionId]/summary.page.tsx

```diff
@@ -134,28 +166,28 @@ export default function SubmissionSummary({
                   section={section}
                   submissionId={grantSubmissionId}
                   mandatoryQuestionId={mandatoryQuestionId}
-                  readOnly={hasSubmissionBeenSubmitted}
+                  readOnly={hasSubmissionBeenSubmitted || closedAndInProgress}
                 />
               ))}
 
               <h1 className="govuk-heading-s" data-cy="cy-download-header">
                 Download a copy of your application
               </h1>
               <p className="govuk-body">
-                You can{' '}
-                <Link
+                You can {/* <Link */}
+                <a
+                  className="govuk-link govuk-link--no-visited-state"
                   href={routes.api.submissions.downloadSummary(
                     grantSubmissionId
                   )}
                 >
-                  <a className="govuk-link govuk-link--no-visited-state">
-                    download a copy of your answers (ZIP)
-                  </a>
-                </Link>{' '}
+                  download a copy of your answers (ZIP)
+                </a>
+                {/* </Link>{' '} */}
```

Comment:
Can we remove this (has been fixed in QA which will eventually get merged into develop)